src/config.c

```diff
@@ -2561,6 +2571,8 @@ standardConfig configs[] = {
     createBoolConfig("disable-thp", NULL, MODIFIABLE_CONFIG, server.disable_thp, 1, NULL, NULL),
     createBoolConfig("cluster-allow-replica-migration", NULL, MODIFIABLE_CONFIG, server.cluster_allow_replica_migration, 1, NULL, NULL),
     createBoolConfig("replica-announced", NULL, MODIFIABLE_CONFIG, server.replica_announced, 1, NULL, NULL),
+    createBoolConfig("aof-disable-auto-gc", NULL, MODIFIABLE_CONFIG, server.aof_disable_auto_gc, 0, NULL, updateAofAutoGCEnabled),
+    createBoolConfig("aof-rewrite-always-fail", NULL, MODIFIABLE_CONFIG, server.aof_rewrite_always_fail, 0, NULL, NULL),
```

Comment:
i think this should not be a config, but rather a DEBUG sub-command (completely unreachable for normal users).
see `SET-ACTIVE-EXPIRE` and alike.
p.s. i haven't looked at the tests yet, but maybe there's a different way to induce a failure without the need of such a config.
maybe by using `rdb-key-save-delay`, and then killing the child from the test (with `kill`)

Reply:
That is what I did before this commit, but I encountered a problem when writing the AOFRW limit test, because I wanted to simulate a scenario where the auto rewrite failed. But after we changed `AOF_REWRITE_LIMITE_THRESHOLD` to 3, it is easy to achieve the goal using kill. So I rolled back the configuration now.

Reply:
FYI: just note i wrote `rdb-key-save-delay` (not `aof-child-rewrite-delay`).

Reply:
I have removed the `aof-child-rewrite-delay` configuration and used the `rdb-key-save-delay` instead.